mlflow/entities/_job.py

```diff
@@ -0,0 +1,74 @@
+from mlflow.entities._mlflow_object import _MlflowObject
+from mlflow.entities._job_status import JobStatus
+
+
+class Job(_MlflowObject):
+    """
+    MLflow entity representing a Job.
+    """
+
+    def __init__(
+        self,
+        job_id: str,
+        creation_time: int,
+        function: str,
+        params: str,
+        status: JobStatus,
+        result: str | None,
+        retry_count: int = 0,
+    ):
+        super().__init__()
+        self._job_id = job_id
+        self._creation_time = creation_time
+        self._function = function
+        self._params = params
+        self._status = status
+        self._result = result
+        self._retry_count = retry_count
+
+    @property
+    def job_id(self) -> str:
+        """String containing job ID."""
+        return self._job_id
+
+    @property
+    def creation_time(self) -> int:
+        """Creation timestamp of the job, in number of milliseconds since the UNIX epoch."""
+        return self._creation_time
+
+    @property
+    def function(self) -> str:
+        """
+        String containing the function full name, in the form of `<module_name>.<function_name>`
+        """
+        return self._function
+
+    @property
+    def params(self) -> str:
+        """
+        String containing the job serialized parameters in JSON format.
+        e.g. `{'a': 3, 'b': 4}` represents there are 2 params `a` with value 3 and
+        `b` with value 4.
```

Comment:
If this is always a JSON object, should the type hint be `dict[str, Any]`?

Reply:
it is a string , value is serialized JSON.